test/e2e/service.go

```diff
@@ -548,6 +548,16 @@ var _ = Describe("Services", func() {
 			i++
 		}
 
+		By("updating service's port " + serviceName + " and reaching it at the same IP")
+		service, err = updateService(f.Client, f.Namespace.Name, serviceName, func(s *api.Service) {
+			s.Spec.Ports[0].Port = 19482
```

Comment:
where did the original port come from? do we have some guarantee this port is different from the original?